From 4de1e2fc78d798c069ab6483b100dd4c0aeb62be Mon Sep 17 00:00:00 2001
From: Richard W.M. Jones <rjones@redhat.com>
Date: Thu, 30 Aug 2012 18:16:18 +0100
Subject: [PATCH 22/32] EPEL 5: podwrapper: Don't use Pod::Man.

This unfortunately means the man pages are generated as plain text.
However that's necessary as Pod::Man does not have an output_string
method.
---
 podwrapper.pl.in |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/podwrapper.pl.in b/podwrapper.pl.in
index 53a13cd..415fa67 100755
--- a/podwrapper.pl.in
+++ b/podwrapper.pl.in
@@ -387,7 +387,7 @@ SUBMAN: {
     package Podwrapper::Man;
 
     use vars qw(@ISA $VERSION);
-    @ISA = qw(Pod::Man);
+    @ISA = qw(Pod::Simple::Text);
     $VERSION = $package_version;
 
     # Override the L<> method.
-- 
1.7.4.1

